src/components/PostPageDetail/index.js

```diff
@@ -80,7 +80,7 @@ const PostPageDetailIdComponent = (props) => {
   const {getTotalReaction, getHeightHeader} = useFeed();
   const [commentContext, dispatchComment] = React.useContext(Context).comments;
   const {comments} = commentContext;
-  const [loadingGetComment, setLoadingGetComment] = React.useState(true);
+  const [loadingGetComment, setLoadingGetComment] = React.useState(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The initial state of `loadingGetComment` has been changed from `true` to `false`. This change might affect the loading behavior of comments. Ensure that this doesn't lead to a situation where the user sees an empty comment section before the comments are loaded.
```diff
-  const [loadingGetComment, setLoadingGetComment] = React.useState(true);
+  const [loadingGetComment, setLoadingGetComment] = React.useState(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->